modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpSymfonyServerCodegen.java

```diff
@@ -244,6 +249,12 @@ public void processOpts() {
             additionalProperties.put(BUNDLE_NAME, bundleName);
         }
 
+        if (additionalProperties.containsKey(BUNDLE_ALIAS)) {
+            this.setBundleAlias((String) additionalProperties.get(BUNDLE_ALIAS));
+        } else {
+            additionalProperties.put(BUNDLE_ALIAS, bundleAlias);
```

Comment:
@HichemChiouchiou  If BundleAlias is not provided, what about defaulting to the logic in line 193 to make the fix backward compatible?
```
        this.bundleAlias = snakeCase(bundleName).replaceAll("([A-Z]+)", "\\_$1").toLowerCase(Locale.ROOT);
```

Reply:
Done!